### ISSUE TITLE
TypeError: Unable to get property 'div' of undefined or null reference when closing modal in ie 11

### ISSUE THREAD

**Issue by dresdor** [VadimDez/ng2-pdf-viewer#354]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [ x ] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

On our app, we display the pdf viewer in a modal.  When the modal is closed, in IE 11, it will generate a console error:

TypeError: Unable to get property 'div' of undefined or null reference

If the pdf viewer component is removed from the modal, there is no error.

The html for the viewer is straightforward:

     <pdf-viewer [src]="pdfSrc"
            [render-text]="true"
            style="display: block;"></pdf-viewer>

The modal code is pretty straightforward as well:

    openDialog() {
      let dialogRef = this.dialog.open(PdfModalComponent, {
        maxWidth: '200em',
        height: '90%',
        data: {}
      });
    }

Where PdfModalComponent is a very slim component containing the pdf viewer html template and the following method:

    public response(resp): void {
      this.dialogRef.close({ data: resp });
    }

Using Version 5.1.1

**Comment by VadimDez** [comment]
What version of `ng2-pdf-viewer` are you using?

**Comment by dresdor** [comment]
@VadimDez  using version 5.1.1

**Comment by subatomicglue** [comment]
I get this with 5.1.1 in chrome if I close (ngIf=false the parent div) too quickly before the PDF is totally loaded...    I am catching the `after-load-complete` and `page-rendered`, not allowing to set the parent div's `ngIf` to false unless those have completed, but still getting the crash/exception.

```
TypeError: Cannot read property 'div' of undefined
    at backtrackBeforeAllVisibleElements (VM474151 pdf_viewer.js:558)
    at getVisibleElements (VM474151 pdf_viewer.js:605)
    at PDFViewer._getVisiblePages (VM474151 pdf_viewer.js:5400)
    at PDFViewer.forceRendering (VM474151 pdf_viewer.js:5080)
    at PDFRenderingQueue.renderHighestPriority (VM474151 pdf_viewer.js:4191)
    at continueRendering (VM474151 pdf_viewer.js:4256)
    at ZoneDelegate.invoke (VM472116 zone.js:388)
```

**Comment by subatomicglue** [comment]
see also: https://github.com/VadimDez/ng2-pdf-viewer/issues/367

**Comment by VadimDez** [comment]
Should be fixed in the latest version `5.3.1`
